pkg/logutil/log.go

```diff
@@ -143,6 +143,27 @@ func stringToLogLevel(level string) log.Level {
 // textFormatter is for compatability with ngaut/log
 type textFormatter struct {
 	DisableTimestamp bool
+	DisableColors    bool
```

Comment:
Can we automatically turn it on when writing to files?

Reply:
Then the rule would be complex.
We can enable color in file, some tool can interpret the color.

Reply:
I think it's not complex. Just check if _cfg.File.Filename_ is set, and then set `DisableColors` to true.

Reply:
The file config should not affect the format, we may log to stdout, then redirect to a file.

Reply:
I know. I'm suggesting disable color when _cfg.File.Filename_ is set, instead of disable it when the log finally goes to some sort of file.

Or we can make color disabled by default, which I can accept, too. pd-server is a service that saves log into files in most scenarios, we should not consider console beauty as first priority.

Reply:
My mistake. The default value is already true now.